DokanNet/NativeStructWrapper.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
+using System.Text;
+
+namespace DokanNet
+{
+    internal class NativeStructWrapper<T> : SafeBuffer where T : class
```

Comment:
If you have time, it would be great to add some doc here (even a small one) since all the library is documented

Reply:
Just fixed some documentation comments.